File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(long long n) {
    vector<int> answer;
    string str = to_string(n);
    for (int i = 0; i < str.size(); i++) {
        answer.emplace_back(str[i]-'0');
    }
    reverse(answer.begin(), answer.end());
    return answer;
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
class Solution {
    public int[] solution(long n) {
        int[] answer = {};
        Long N = n;
        String s = N.toString();
        answer = new int[s.length()];
        for (int i = 0; i < s.length(); i++) {
            answer[s.length()-1-i] = s.charAt(i) - '0';
        }
        return answer;
    }
}